app/lib/MyAccountsStore.js

```diff
@@ -99,7 +99,33 @@ export default class MyAccountsStore {
           }
         }
       })
+      .then(account => {
+        // add a reference to the folder in the konnector
```

Comment:
I think adding doc refernece will have to be a cozy client method in the future.